### PR TITLE
CASSANDRA-16775 Reduce the log level on "expected" repair exceptions

### PR DESCRIPTION
**Pull request by maedhroz**

### CODE REVIEW

File: src/java/org/apache/cassandra/repair/NoSuchRepairSessionException.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.repair;
+
+import java.util.UUID;
+
+public class NoSuchRepairSessionException extends Exception
```
Comment (jmckenzie-dev) [inline src/java/org/apache/cassandra/repair/NoSuchRepairSessionException.java:23]: Recommend a bread crumb comment as to why "The repair session isn't there" translates into "the repair session failed".

**Review by jmckenzie-dev — APPROVED** [review]
Had to stare at ValidationTask for a bit to really convince myself we'd covered the flow completely with it. Think it's good.

Barring the 1 nit on a comment, +1

**Comment by maedhroz** [comment]
https://app.circleci.com/pipelines/github/maedhroz/cassandra?branch=CASSANDRA-16775

**Comment by maedhroz** [comment]
see https://github.com/apache/cassandra/commit/f49c29ee8c7374a05e32be01cb0046236d5309e6

**Pull request closed without merging**